submissions/cumiOyemike/react/to-do-app/src/components/Todos/Todos.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import './Todos.css';
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+
+class Todos extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            searchTerms: [],
+            newItem: "",
+            completed: false,
+            display: 'none',
+        };
+    }
+
+    updateInput = (key, value) => {
+        this.setState({ [key]: value });
+    }
+
+    addItem = () => {
+        const newItem = {
+            id: 1 + Math.random(),
+            completed: this.state.completed,
+            value: this.state.newItem.slice()
+        }
+
+        const searchTerms = [...this.state.searchTerms];
+        searchTerms.push(newItem);
+        this.setState({ searchTerms, newItem: "" });
+
+        this.state.display = 'inline';
```

Comment:
You are updating state directly. You shouldn't. You can do this instead `this.setState({display: 'inline})` @CtripleU